photo-album/index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../reset.css" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Synthia's Profile Pic Collection</title>
+  </head>
+  <body>
+    <header class="main" id="top">
+      <h1>Synthia's Profile Pic Collection📸</h1>
+    </header>
+
+    <main>
+      <section class="preview">
+        <img
+          src="images/dark/d858d8ef-0140-4f13-aae5-cec2b183683d.jpeg"
+          alt="preview profile pic no.1"
+        />
+        <img
+          src="images/pattern/more blue for ya.jpeg"
+          alt="preview profile pic no.2"
+        />
+        <img
+          src="images/cute/Anything Muppets.jpeg"
+          alt="preview profile pic no.3"
+        />
+        <img
+          src="images/cute/230d9fb3-aa10-44e9-9c1f-70d49f32828f.jpeg"
+          alt="preview profile pic no.4"
+        />
+        <img
+          src="images/cute/Twitter Turned This Sad Kermit Into An Emotional Roller Coaster No One Asked For.jpeg"
+          alt="preview profile pic no.5"
+        />
+        <img src="images/pattern/_.jpeg" alt="preview profile pic no.6" />
+        <img
+          src="images/cute/Spotify covers!.jpeg"
+          alt="preview profile pic no.7"
+        />
+        <img
+          src="images/dark/— 𝐢’𝐦 𝐣𝐮𝐬𝐭 𝐲𝐨𝐮𝐫 𝐩𝐫𝐨𝐛𝐥𝐞𝐦.jpeg"
+          alt="preview profile pic no.8"
+        />
+        <img
+          src="images/pattern/824f1365-e9a9-4a37-ae8f-41bc638362ec.jpeg"
+          alt="preview profile pic no.9"
+        />
+        <img
+          src="images/cute/67377a32-e8ca-42f4-9929-f200a9c84ace.jpeg"
+          alt="preview profile pic no.10"
+        />
+      </section>
+      <nav>
+        <a href="#cute"> Cute Profile Pics🤍</a>
+        <a href="#dark"> Dark Profile Pics👻</a>
+        <a href="#pattern"> Patterned Profile Pics🎨</a>
+      </nav>
```

Comment:
```suggestion
      <nav>
        <ul>
        <li><a href="#cute"> Cute Profile Pics🤍</a></li>
        <li><a href="#dark"> Dark Profile Pics👻</a></li>
        <li><a href="#pattern"> Patterned Profile Pics🎨</a></li>
         </ul>
      </nav>
```
요러면 더 시맨틱해질 수 있겠네요!-!